obj/gebaeude.h

```diff
@@ -127,6 +127,9 @@ class gebaeude_t : public obj_t, sync_steppable
 
 	bool is_building_of_city() const { return is_city_building() || is_headquarter() || is_monument(); }
 
+	// check broken tiles
+	bool is_broken_building(); 
```

Comment:
この関数は状態を変更しないので、 `const` でそれを明示してください。
```suggestion
	bool is_broken_building() const; 
```